config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -2006,6 +2006,10 @@
 # ALTERNATELY, use the comma seperated list form
 # prevent.public.announcements=project,portfolio
 
+# Decide whether to use the site title or the short description in the subject when notifying an announcement by email.
+# DEFAULT: false
+# sakai.announcement.email.subject.short.description=true
```

Comment:
Really minor, I'd like to have a different property name here, something like:
announcement.email.use.short.description=true

Totally minor, let's see what others think.